dist/fixed-data-table-base.css

```diff
@@ -1,5 +1,5 @@
 /**
- * FixedDataTable v0.8.6 
+ * FixedDataTable v0.8.8 
```

Comment:
We don't want to modify these files directly.  Could you revert your changes to the dist directory for the PR?